src/styles/app.styl

```diff
@@ -32,6 +32,7 @@ body
     mix-blend-mode multiply
     overflow-y auto
     .corner
+        z-index $nav-index
```

Comment:
you should avoid the use of the stylus var.   var(--zIndex-nav) should do the trick ;)